surface/include/pcl/surface/3rdparty/poisson4/polynomial.hpp

```diff
@@ -296,14 +301,14 @@ namespace pcl
         }
       }
     }
-    template< >
+    template< > inline
     Polynomial< 0 > Polynomial< 0 >::BSplineComponent( int i )
     {
       Polynomial p;
       p.coefficients[0] = 1.;
       return p;
     }
-    template< int Degree >
+    template< int Degree > inline
```

Comment:
Marking this inline is debatable for me. But since it's just gut feeling, I will accept the change.